bin/jgrep

```diff
@@ -39,6 +41,7 @@ def do_grep(json, expression)
                     result.map{|x| puts x unless x.nil?}
```

Comment:
and another just above here! It's used with `-s` in circumstances with an array.